addons/viewdistance/XEH_preInit.sqf

```diff
@@ -6,4 +6,15 @@ PREP_RECOMPILE_START;
 #include "XEH_PREP.hpp"
 PREP_RECOMPILE_END;
 
+#include "initSettings.sqf"
+
+if (GVAR(enabled) && hasInterface && !is3DEN) then {
```

Comment:
Should be executed on `CBA_settingsInitialized` event. Otherwise the server overwrite setting might not be received yet and it will have no effect.

Reply:
Good to know